content/custom-decorators.md

```diff
@@ -115,8 +115,11 @@ Let's define a decorator that takes a property name as key, and returns the asso
 @@filename(user.decorator)
 import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 
+// This should be a real class/interface representing a user entity
+export type UserEntity = any;
+
 export const User = createParamDecorator(
-  (data: string, ctx: ExecutionContext) => {
+  (data: keyof UserEntity, ctx: ExecutionContext) => {
```

Comment:
Instead of updating the example we should rather add a hint blockquote saying that keyof could be used (to achieve better type-safety)

Reply:
sure will update that